client/src/components/Form/FormCard.test.js

```diff
@@ -0,0 +1,31 @@
+import { mount } from "@vue/test-utils";
+import { getLocalVue } from "jest/helpers";
+import FormCard from "./FormCard";
+
+jest.mock("app");
```

Comment:
Why?

Reply:
Thanks for catching it, this is a left over from a test example elsewhere.